rusk/src/lib/builder/node.rs

```diff
@@ -46,6 +51,12 @@ pub struct RuskNodeBuilder {
     state_dir: PathBuf,
 
     http: Option<HttpServerConfig>,
+
+    #[cfg(feature = "archive")]
+    archive_channel: (
+        Option<tokio::sync::mpsc::Sender<ArchivalData>>,
+        Option<tokio::sync::mpsc::Receiver<ArchivalData>>,
+    ),
```

Comment:
Shouldn't this be `Option<(Sender, Receiver)>` instead? Or even simply `(Sender, Receiver)`. Why is it an `Option` in the first place?

Reply:
Structuring this differently might help in justifying downstream `expect`s, or even downright remove their need.